consensus/src/round_manager.rs

```diff
@@ -539,17 +543,21 @@ impl RoundManager {
     pub async fn process_vote_msg(&mut self, vote_msg: VoteMsg) -> anyhow::Result<()> {
         trace_code_block!("round_manager::process_vote", {"block", vote_msg.proposed_block_id()});
         // Check whether this validator is a valid recipient of the vote.
-        self.ensure_round_and_sync_up(
-            vote_msg.vote().vote_data().proposed().round(),
-            vote_msg.sync_info(),
-            vote_msg.vote().author(),
-            true,
-        )
-        .await
-        .context("[RoundManager] Stop processing vote")?;
-        self.process_vote(vote_msg.vote())
+        if self
```

Comment:
I agree that reporting errors for the f votes that occur after 2f+1 doesn't make sense.  

We shouldn't expect this for proposals though right?  I.e. For debugging, would it be useful to keep some error message that we received an invalid stale proposal?

Reply:
it looks similar to me, if one node is stale it'd broadcast stale proposals. it's just not as expected as the f votes

Reply:
I'm a little worried about losing hints about misbehaving nodes to remove these expected error messages - more so in the proposal case instead of the voting case (since its expected).  

I.e. How would we determine that a node is accidentally flooding others with bad proposals or votes?

Reply:
a misbehaving node can flood others with the exact votes that we ignore too, I don't think the added value of detecting stale messages worth specializing. if things goes wrong, there's enough logging to debug.

Reply:
Which logs are you referring to specifically for detecting floods?